src/igvfd/tests/test_upgrade_page.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+
+def test_page_upgrade_2_3(upgrader, page_v2):
+    value = upgrader.upgrade('page', page_v2, current_version='2', target_version='3')
```

Comment:
I think upgrade has to drop properties and then the test has to check they were dropped

Reply:
OK. We talked about this in the meeting and somehow I came away with the impression I didn’t have to do that since no page objects exist in igvfd.